src/main/events/library/open-game-installer.ts

```diff
@@ -29,15 +44,22 @@ const openGameInstaller = async (
     return true;
   }
 
+  if (fs.lstatSync(gamePath).isFile()) {
+    return executeGameInstaller(gamePath);
+  }
+
   const setupPath = path.join(gamePath, "setup.exe");
-  if (!fs.existsSync(setupPath)) {
-    shell.openPath(gamePath);
-    return true;
+  if (fs.existsSync(setupPath)) {
+    return executeGameInstaller(setupPath);
   }
 
-  if (process.platform === "win32") {
-    shell.openPath(setupPath);
-    return true;
+  const gamePathFileNames = fs.readdirSync(gamePath);
+  const gameAlternativeSetupPath = gamePathFileNames.find(
+    (fileName: string) => path.extname(fileName).toLowerCase() === ".exe"
+  );
+
+  if (gameAlternativeSetupPath) {
+    return executeGameInstaller(path.join(gamePath, gameAlternativeSetupPath));
```

Comment:
What do you think about this? If for some reason there is more than 1 .exe the code won't select an arbitrary one.
The rest of the login remains the same


```suggestion
  const gameAlternativeSetupPath = gamePathFileNames.filter(
    (fileName: string) => path.extname(fileName).toLowerCase() === ".exe"
  );

  if (gameAlternativeSetupPath.length == 1) {
    return executeGameInstaller(path.join(gamePath, gameAlternativeSetupPath[0]));
```

Reply:
Yeah, we can do this.